### DISCUSSION TITLE
help is needed

### DISCUSSION THREAD

**Discussion by subsubsubsubsubsub** [Serene-Arc/bulk-downloader-for-reddit#956] [Q&A]
describe how to make a profile backup in the form of links. saved likes and diss and subscriptions

**Comment by Serene-Arc** [comment]
The information you require is the README. All of the options are detailed there.

**Comment by subsubsubsubsubsub** [comment]
I don’t understand how to do this and I’m writing

**Reply by Serene-Arc** [reply]
What part of the README do you not understand?
